plugins/source/github/resources/services/actions/workflow_jobs.go

```diff
@@ -36,7 +36,7 @@ func fetchWorkflowJobs(ctx context.Context, meta schema.ClientMeta, resource *sc
 	listOpts := &github.ListOptions{PerPage: 100}
 	listWorkflowJobOptions := &github.ListWorkflowJobsOptions{Filter: "all", ListOptions: *listOpts}
 	for {
-		workflowJobs, resp, err := c.Github.Actions.ListWorkflowJobs(ctx, c.Org, *repo.Name, *run.ID, listWorkflowJobOptions)
+		workflowJobs, resp, err := c.Github.Actions.ListWorkflowJobs(ctx, *repo.Owner.Login, *repo.Name, *run.ID, listWorkflowJobOptions)
```

Comment:
This is how we do it in most places (access the owner via the repo). It should be the same as accessing the org directly